src/NUnitTestAdapter/NUnit3TestExecutor.cs

```diff
@@ -266,7 +266,13 @@ private void RunAssembly(string assemblyName, TestFilter filter)
                     }
                 }
                 else
-                    TestLog.Info("NUnit failed to load " + assemblyName);
+                {
+                    var msgNode = loadResult.SelectSingleNode("properties/property[@name='_SKIPREASON']");
+                    if (msgNode != null && (new[] { "contains no tests", "Has no TestFixtures" }).Any(msgNode.GetAttribute("value").Contains))
+                        TestLog.Info("NUnit couldn't find any tests in " + assemblyName + ". (Probably an assembly dependency problem in your projects.");
```

Comment:
The `(Probably an assembly dependency problem in your projects.` bothers me as there are several reasons that this could be the case. I would prefer to leave that part out or at least change it to Possibly. At the very least, the trailing `)` is missing in the message.